adapters/pom.xml

```diff
@@ -26,6 +26,6 @@
 
   <modules>
     <module>ddf-adapter</module>
-    <module>webhdfs-adapter</module>
+<!--    <module>webhdfs-adapter</module>-->
```

Comment:
❓ Why did we have to remove this?

Reply:
We had removed it from the feature files when upgrading to DDF 2.25.x. It will take additional work to get the webhdfs adapter working with 2.25.x and address owasp findings.